apps/site-nft/src/components/HomePage/index.tsx

```diff
@@ -0,0 +1,8 @@
+import { AppContainer } from 'components/AppContainers';
+import * as React from 'react';
+
+export const HomePage = () => (
+  <AppContainer shadow>
+    HOME PAGE
```

Comment:
I am feeling anxious seeing text not being translatable even if I know its temporary 😂

Reply:
Sorry!  The other pages will hopefully make you feel a bit better!